public/Miscellaneous/printYoutube.js

```diff
@@ -1,35 +1,78 @@
-if (!window["youtubeIFrame"]) {
+﻿if (!window["youtubeIFrame"]) {
```

Comment:
Could you run this file through the VS Code formatter so that it looks nicer? It may be called "beautify this file".